src/Components/Buttons.mdx

```diff
@@ -19,92 +19,119 @@ import { Playground } from 'docz';
+## General Usage Guidelines
 
-## Usage
+- Buttons should be used for important action items like submitting forms, saving pages, or completing purchases
+- Buttons can also indicate the beginning of a new task or the next steps in a multi-step process
+- Buttons can also be used when creating new entities like new sites, new blog posts, or new products
 
-Use any of the available button style types to quickly create a styled button. Just modify the bsStyle prop.
+### Label
 
-**Primary** - Primary buttons draw the users attention to a specific action
+Button labels should be verbs that connote action like Save, Delete, or Complete Purchase. 
+
+### Standard (Primary) Button 
+
+- The strongest calls to action on the page.
+- Try to use only one primary call to action per page.
+- Not all pages require a primary call to action.
 
 <Playground>
   <ButtonToolbar>
-    <Button bsStyle="primary">Primary</Button>
+    <Button bsStyle="primary">Standard</Button>
+    <Button bsStyle="primary" active>Active</Button>
+    <Button bsStyle="primary" disabled>Disabled</Button>
   </ButtonToolbar>
 </Playground>
 
-**Secondary** - Secondary buttons emphasize actions that are less important
+### Outline (Secondary) Button 
+
+- Use when the pages primary action is not obvious or when there are complimentary actions the user can perform.
+- There can be multiple outline/secondary buttons on a single page.
 
 <Playground>
   <ButtonToolbar>
-    <Button>Secondary</Button>
+    <Button>Outline</Button>
+    <Button active>Active</Button>
+    <Button disabled>Disabled</Button>
   </ButtonToolbar>
 </Playground>
 
-**Destruction** - Destructive buttons express caution to users and are tied to actions like delete or deactivate
+### Link Button
+
+- Use when trying to give an action the least possible emphasis.
+- Use when there are many actions available and you are trying not to saturate the page with buttons. 
+- Link buttons are not hyperlinks in a pararaph of text, they are to be used like buttons.
+- There can be multiple link buttons on a single page.
 
 <Playground>
   <ButtonToolbar>
-    <Button bsStyle="danger">Destructive</Button>
+    <Button bsStyle="link">Standard</Button>
+    <Button bsStyle="link" active>Active</Button>
+    <Button bsStyle="link" disabled>Disabled</Button>
   </ButtonToolbar>
 </Playground>
 
-**Text** - Text buttons emphasize actions that are the least important
+### Destructive Button
+
+- Use whenever a destructive action is present on a screen. 
+- These buttons are typically used in modals whereby the user must confirm the destructive action.
 
 <Playground>
   <ButtonToolbar>
-    <Button bsStyle="link">Link</Button>
+    <Button bsStyle="danger">Danger</Button>
+    <Button bsStyle="danger" active>Active</Button>
+    <Button bsStyle="danger" disabled>Disabled</Button>
   </ButtonToolbar>
 </Playground>
 
-**Small** - Small buttons \_\_\_
+### Button Sizing
+
+<Playground>
+  <ButtonToolbar>
```

Comment:
Minor: You could add `className="mb2"` to each ButtonToolbar here to add some space so they aren't touching.